torchelie/recipes/upsamplinggan.py

```diff
@@ -192,7 +192,6 @@ class PerceptualDiscriminator(nn.Module):
     @tu.experimental
     def __init__(self, n_downscales=4):
         super().__init__()
-        layers = tu.FrozenModule(tch.models.PerceptualNet(layers))
```

Comment:
Disregard everything in this file.

Reply:
Should I take it as a request to keep this line in and revert this change from the PR?